Source/Particles/PhysicalParticleContainer.cpp

```diff
@@ -1109,9 +1109,9 @@ PhysicalParticleContainer::Evolve (int lev,
                 WARPX_PROFILE_VAR_STOP(blp_fg);
 
                 //
-                // Current Deposition (only needed for electromagnetic solver)
+                // Current Deposition
```

Comment:
I remove the comment on the electromagnetic solver, since - for the new "multi-J" mode, we would skip this also for the electromagnetic solver. (but then call it separately later)